src/classes/TDTM_TriggerHandler.cls

```diff
@@ -197,20 +196,7 @@ public class TDTM_TriggerHandler {
     * @param dmlWrapper The class that contains the records that need to have DML performed on.
     * @return void
     */
-    public static void processDML(TDTM_Runnable.DmlWrapper dmlWrapper) {
-        processDMLWithRollback(dmlWrapper, true);
-    }
-    /*******************************************************************************************************
-    * @description Performs pending DML operations, on the records stored in the DMLWrapper that it gets as parameter.
-    * We want this method to be static so that it can be called from the future method in TDTM_Runnable.
-    * @param dmlWrapper The class that contains the records that need to have DML performed on.
-    * @param withRollback True if errors should cause rollback, false if not.
-    * @return Errors Object with list of Error objects
-    */
-    public static ERR_Handler.Errors processDMLWithRollback(TDTM_Runnable.DmlWrapper dmlWrapper, boolean withRollback) {
-        //Create transaction
-        Savepoint sp = Database.setSavepoint();
```

Comment:
Saving one DML operation (setting the savepoint), until we do the error handling re-work.